CHANGELOG.md

```diff
@@ -4,13 +4,19 @@ IMPROVEMENTS:
 * `data/policy_document`: Add support for `patch` capability for vault-1.9+. ([#1238](https://github.com/hashicorp/terraform-provider-vault/pull/1238))
 * `resource/database_secret_backend_connection`: Add support for InfluxDB connections ([#1121](https://github.com/hashicorp/terraform-provider-vault/pull/1121))
 * `resource/generic_secret`: Add support for deleting all version data for a KV-V2 secret ([#1254](https://github.com/hashicorp/terraform-provider-vault/pull/1254))
+* `resource/jwt_auth_backend`: Add support for `oidc_response_mode`,`oidc_response_types` and `namespace_in_state` fields ([#1244](https://github.com/hashicorp/terraform-provider-vault/pull/1244))
+* `resource/vault_identity_entity_alias`: Add `custom_metadata` support for entity aliases ([#1235]https://github.com/hashicorp/terraform-provider-vault/pull/1235)
+* `provider`: Add `create_intermediate_child_token` optional configuration setting to control the use of ephemeral child tokens ([#775]https://github.com/hashicorp/terraform-provider-vault/pull/775)
```

Comment:
I think we can move this to the `FEATURES` section.

Reply:
Removed in this PR since I now realize you will be adding it in yours :)